WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -4,7 +4,7 @@ import WordPressShared
 
 
 class PageListViewController: AbstractPostListViewController, UIViewControllerRestoration {
-
+    fileprivate static let HTTPErrorCodeForbidden = Int(403)
```

Comment:
This line probably isn't needed now. :)

Reply:
Yep! Good catch! Fix it!

Reply:
@aerych what do you think about `bind` or `binding`?

Reply:
Arg! I missed this comment until just now.  Sorry! 
`bind` or `binding` instead of `completion`? 🤔  I dunno... that's more reflective of what the block passed by PageListViewController does.  The block is executed after the restore is complete so `completion` kind of makes sense except there is other code still to be executed so I think that name gives the wrong impression.  Maybe i'm just being pendantic 😬.   
I'd say something like `onRestore` but that doesn't sound very Swifty.  Maybe `restoreComplete`? 🤷‍♂️ 
Honestly we could just :shipit: and worry about the naming another day.